frontend/components/tableDisplay/tableNameColumn.tsx

```diff
@@ -0,0 +1,19 @@
+// Creates ReactNode instead of ReactElement
+// so that petTableDisplay.tsx can render in name column.
+
+import Image from 'next/image'
+
+const TableName = (props) => {
+	const { name, image } = props;
+
+	let element: JSX.Element;
+
+  return (
+    <div>
+      {image ? (<img src={image} alt="pet-image" style={{height: "60px", width: "auto"}} />): ""}
```

Comment:
would be nice to use `<Image>` since it is imported anyways and seems useful.

https://nextjs.org/docs/api-reference/next/image

Reply:
at the start of the project we did, but there was a recommendation to not use `<Image>`
[this commit](https://github.com/bettersg/PawScore/commit/f65c0404d4c5d89d26cca37d56b93d30dc2f01f2)

Reply:
@chunqi can advise please?

Reply:
@stonefruit I suggest using the regular `<img>` as the Next.js `<Image>` component will enable server side image optimisation, which prevents a static HTML export of the frontend.

https://nextjs.org/docs/basic-features/image-optimization
https://nextjs.org/docs/advanced-features/static-html-export